bsp/src/org/jetbrains/bsp/project/resolver/BspProjectResolver.scala

```diff
@@ -79,8 +79,13 @@ class BspProjectResolver extends ExternalSystemProjectResolver[BspExecutionSetti
               List.empty[ScalacOptionsItem]
             }
 
+            val testClasses = data.scalaTestClasses.map(_.getItems.asScala).getOrElse {
```

Comment:
It makes sense to load the classes if we want to select a group of them with a regex. However should we store them in BspMetadata then (or any other way of persistence) ? If yes, then how can that be done. The existing code will not allow to do that easily, since it computes the data all together during the import.

Reply:
We can store them separately in a PersistentStateComponent. See eg `org.jetbrains.sbt.project.module.SbtModule`